docs/run_hydradx_collator.md

```diff
@@ -0,0 +1,115 @@
+---
+id: run-collator
+title: Run your own collator
+---
+
+This is a step-by-step how-to so you can get your HydraDX collator up and running. In this guide, we use Ubuntu 20.04 LTS
+
+## Create a technical `hydra` user and add it to Sudoers
+
+```bash
+sudo adduser hydra
+sudo usermod -aG sudo hydra
+su - hydra
+```
+
+## Download and configure the `hydradx` binary
+
+Pick a 12.x release, we are using `v12.1.0` from our assets repository:
+
+```bash
+wget https://github.com/galacticcouncil/HydraDX-node/releases/download/v12.1.0/hydradx
+sudo mv hydradx /usr/local/bin
+sudo chmod +x /usr/local/bin/hydradx
+sudo chown hydra:hydra /usr/local/bin/hydradx
+
+```
+
+## Command to run your collator
+
+Best is to run your collator as a `service` using `systemctl`. To do so, create a file, namely `hydradx-collator.service` under `/etc/systemd/system/hydradx-collator.service`:
+
+```bash
+sudo vim /etc/systemd/system/hydradx-collator.service
+```
+
+Then paste the following:
+
+```bash
+[Unit]
+Description=hydradx validator
+[Service]
+Type=exec
+User=hydra
+ExecStart=/usr/local/bin/hydradx \
+    --name YOUR_COLLATOR_NAME \
+    --collator \
+    -- \
+    --execution wasm \
+    --prometheus-external \
```

Comment:
prometheus external has to be in parachain part if you want to export parachain start and not relaychain, it is worth noting that you have to set up separate port when you want to export both

Reply:
Done.